src/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="AvoidStarImport">
+            <property name="severity" value="warning" />
+        </module>
+        <module name="IllegalImport">
+            <property name="illegalPkgs" value="java.util.logging"/>
+        </module>
```

Comment:
what about adding the following?
```<module name="RegexpMultiline">
    <property name="format" value="\r\n"/>
    <property name="message" value="Do not use Windows line endings"/>
</module>
```
I guess it would depend on if the infra has a badly configured git on windows agents or not (`core.autocrlf=true` is evil)